shell/platform/windows/text_input_plugin.cc

```diff
@@ -180,6 +241,22 @@ void TextInputPlugin::HandleMethodCall(
   result->Success();
 }
 
+Rect TextInputPlugin::GetCursorRect() const {
+  Rect rect = {composing_rect_.left() * editabletext_transform_[0][0] +
+                   composing_rect_.top() * editabletext_transform_[1][0] +
+                   editabletext_transform_[3][0] + composing_rect_.width(),
+               composing_rect_.left() * editabletext_transform_[0][1] +
+                   composing_rect_.top() * editabletext_transform_[1][1] +
+                   editabletext_transform_[3][1] + composing_rect_.height(),
+               composing_rect_.right() * editabletext_transform_[0][0] +
+                   composing_rect_.bottom() * editabletext_transform_[1][0] +
+                   editabletext_transform_[3][0] + composing_rect_.width(),
+               composing_rect_.right() * editabletext_transform_[0][1] +
+                   composing_rect_.bottom() * editabletext_transform_[1][1] +
+                   editabletext_transform_[3][1] + composing_rect_.height()};
```

Comment:
Why is width and height being added to all four components here? (I assume I'm just not understanding the math of the transform, but it seems very strange to me.)

Reply:
Rewrote in terms of origin/size, which matches the existing Linux implementation better as well.